### DISCUSSION TITLE
hud lässt sich nicht öffnen

### DISCUSSION THREAD

**Discussion by coolcook08** [Courseplay/Courseplay_FS22#2496] [Q&A]
Hallo zusammen 
habe folgende probleme zum einen lässt sich das hud nicht öffnen zum anderen habe ich keinen mauszeiger im spiel

**Comment by schwiti6190** [comment]
Hi @coolcook08, 
probiere mal die 3 Punkte hier: https://github.com/Courseplay/Courseplay_FS22/wiki/FAQ#courseplay-hud-isnt-opening-with-a-right-mouse-click

**Comment by coolcook08** [comment]
Hat leider nicht s gebracht Von meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: David Schwietering ***@***.***> Datum: 27.04.23  20:55  (GMT+01:00) An: Courseplay/Courseplay_FS22 ***@***.***> Cc: coolcook08 ***@***.***>, Mention ***@***.***> Betreff: Re: [Courseplay/Courseplay_FS22] hud lässt sich nicht öffnen (Discussion #2496) 
Hi @coolcook08,
probiere mal die 3 Punkte hier: https://github.com/Courseplay/Courseplay_FS22/wiki/FAQ#courseplay-hud-isnt-opening-with-a-right-mouse-click

**Reply by Tensuko** [reply]
logfile bitte anhängen
